service/src/test/java/uk/gov/hmcts/reform/fpl/service/payment/PaymentServiceTest.java

```diff
@@ -300,4 +352,16 @@ void resetInvocations() {
             .service(FPL)
             .siteId("TEST_SITE_ID");
     }
+
+    private CreditAccountPaymentRequest.CreditAccountPaymentRequestBuilder testNoCaseNumberPaymentRequestBuilder() {
```

Comment:
Probably don't need to return builder here I don't think. Could we not just take `amount` and `fees` as param and return `CreditAccountPaymentRequest`

Reply:
good point.. one more generic addition to this based on Adam's comment,  I have taken entire FeeDto which we can use for fees, amount, withc2 and withoutc2 all the cases